examples/nlp/language_modeling/tuning/megatron_gpt_qat.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Now it would be good to rename megatron_gpt_quantization.py to megatron_gpt_ptq.py :p (separate mr, outside of scope here)